elastic/apm-server
# Universal Profiling: avoid K/V document delete on duplicates

## Motivation/summary

The Universal Profiling intake currently generates tombstones when the same stacktrace/stackframe document is ingested from 2 different agents.
We initially understood that an `op_type=index` is a no-op if the document _id is the same and the document has is not changed, but this is not true.

## Checklist


For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**? ✅ 
- Is its use being published in **telemetry** to enable product improvement? ⛔ 
- Have system tests been added to avoid regression? ✅ 

## How to test these changes

Enable the profiling intake and send some data through it via an Universal Profiling agent: there should be no deleted docs in the `profiling-stacktraces` and `profiling-stackframes` reading their status via the `_cat` API.

## Related issues
